proto/buf/alpha/registry/v1alpha1/labels.proto

```diff
@@ -60,8 +60,6 @@ message MoveLabelRequest {
   LabelName label_name = 1;
   optional LabelValue from = 2;
   LabelValue to = 3;
-  string repository_owner = 4;
-  string repository_name = 5;
```

Comment:
Should these be moved to the `CreateLabelRequest` instead?

Reply:
`CreateLabelRequest` also takes a `LabelValue` in the form of a commit, so we are actually resolving the repo through the commit in that RPC already.

Reply:
Correct, and I assume the implementation checks the authz permissions for the parent repo of the matched commit?

Reply:
Yes, correct.